packages/react-table/src/components/Table/EditableSelectInputCell.tsx

```diff
@@ -1,31 +1,17 @@
 import * as React from 'react';
 import { css } from '@patternfly/react-styles';
-import { Select, SelectOptionObject, SelectProps } from '@patternfly/react-core';
+import { Select, SelectOptionObject } from '@patternfly/react-core';
 import inlineStyles from '@patternfly/react-styles/css/components/InlineEdit/inline-edit';
 import formStyles from '@patternfly/react-styles/css/components/Form/form';
+import { EditableSelectInputProps } from './base';
```

Comment:
These do not appear to be included in the md file.  Can we do that so they show up in the docs.